easybuild/easyconfigs/p/pytest/pytest-7.2.2-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,72 @@
+easyblock = 'PythonBundle'
+
+name = 'pytest'
+version = '7.2.2'
+
+homepage = 'https://docs.pytest.org/en/latest/'
+description = """The pytest framework makes it easy to write small,
+readable tests, and can scale to support complex functional testing for
+applications and libraries."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+dependencies = [
+    ('binutils', '2.37'),
```

Comment:
I may be overlooking something here, but why is binutils a dependency? It's typically only a build dependency. Or is it because `pytest` also invokes the linker to link e.g. tests?

Reply:
Ah, good point, this is a library not a binary...

Reply:
Ah yes, no binaries involved here, only scripts, and Python includes binutils as a runtime dep already.